features/wasm-string-builtins.yml

```
@@ -0,0 +1,6 @@
+name: JavaScript string builtins (WebAssembly)
```

I think we should avoid "JavaScript" in the feature name here to avoid conflation with actual JavaScript features.

```suggestion
name: String builtins (WebAssembly)
```

---

I couldn't decide if I wanted "JavaScript" here or not given it is String builtins taken from the JavaScript String builtin but I guess it is fine without.